manifest/service.go

```diff
@@ -82,11 +82,19 @@ func (s *Service) Proxies(app string) []Proxy {
 		if p.External() {
 			name := fmt.Sprintf("%s-%s-proxy-%d", app, s.Name, p.Balancer)
 
+			networkName := ""
```

Comment:
Sure, will add that. I used same logic as for launching the service containers themselves: https://github.com/convox/rack/blob/master/manifest/process.go#L56 It picks the last defined network, yes. According to documentation, `convox` supports only single network, no? https://convox.com/docs/docker-compose-file/